code/modules/surgery/organs/heart.dm

```diff
@@ -455,6 +455,9 @@
 	///Is the crystal still being built? True by default, gets changed after a timer.
 	var/being_built = TRUE
 
+/obj/structure/ethereal_crystal/relaymove()
+	return
```

Comment:
can you talk me through this being here

Reply:
This prevents the "you can not move while buckled to X" message it does nothing else.

Reply:
ok ping me when u fix merge conflicts @cobby.

Reply:
im sorry real cobby i accidentally pigned you